server/server.go

```diff
@@ -249,6 +250,16 @@ func New(c *Config) (*Server, error) {
 	}))
 	mux.Handle(pat.New("/details/*"), details)
 
+	simulateHandler := handler.Simulate{Base: basePolicyHandler}
+	simulateAPI := goji.SubMux()
+	simulateAPI.Use(middleware.RepoAuth(&middleware.GitHubTokenValidator{ClientCreator: cc}))
+	simulateAPI.Handle(pat.Get("/status"), &handler.SimulateStatus{
```

Comment:
As mentioned in the handler, I think this should be a POST request. I also don't think the `/status` suffix is necessary - to me, it is implied by the rest of the API.